src/cljs/main/broadfcui/header.cljs

```diff
@@ -41,7 +41,8 @@
    {:label [:div {:style {:borderRadius 2
                           :backgroundColor (:background-light style/colors)
                           :color "#000" :textDecoration "none"
-                          :padding "0.5rem" :border style/standard-line}}
+                          :padding "0.5rem" :border style/standard-line}
+                  :data-test-id "accountDropdown"}
```

Comment:
I can change this here.

In profile.cljs, I reused `key` which is already camelCase, e.g. "firstName", instead of adding a parameter to `:render-field` and `:render-nested-field`. Do you think it's worth adding a parameter specifically for the `data-test-id` property so that the two are decoupled? That would have the happy side-effect that the value for `data-test-id` and can be kebab-case.